tests/rubicon/optimism/rubicon_optimism_offers_test.sql

```diff
@@ -0,0 +1,272 @@
+        END AS blockchain_test,
+
+        -- test the project 
+        CASE 
+            WHEN LOWER(test.project) = LOWER(actual.project) THEN TRUE
+            ELSE FALSE
+        END AS project_test,
+
+        -- test the version 
+        CASE 
+            WHEN test.version = actual.version THEN TRUE
+            ELSE FALSE
+        END AS version_test,
+
+        -- test the block_date
+        CASE
+            WHEN test.block_date = actual.block_date THEN TRUE
+            ELSE FALSE
+        END AS block_date_test,
+
+        -- test the block_time
+        CASE
+            WHEN test.block_time = actual.block_time THEN TRUE
+            ELSE FALSE
+        END AS block_time_test,
+
+        -- test the block_number
+        CASE
+            WHEN test.block_number = actual.block_number THEN TRUE
+            ELSE FALSE
+        END AS block_number_test,
+
+        -- test the tx_index
+        CASE
+            WHEN test.tx_index = actual.tx_index THEN TRUE
+            ELSE FALSE
+        END AS tx_index_test,
+
+        -- test the evt_index
+        CASE
+            WHEN test.evt_index = actual.evt_index THEN TRUE
+            ELSE FALSE
+        END AS evt_index_test,
+
+        -- test the sell_token_symbol
+        CASE
+            WHEN LOWER(test.sell_token_symbol) = LOWER(actual.sell_token_symbol) THEN TRUE
+            ELSE FALSE
+        END AS sell_token_symbol_test,
+
+        -- test the buy_token_symbol
+        CASE
+            WHEN LOWER(test.buy_token_symbol) = LOWER(actual.buy_token_symbol) THEN TRUE
+            ELSE FALSE
+        END AS buy_token_symbol_test,
+
+        -- test the sell_token_address
+        CASE
+            WHEN LOWER(test.sell_token_address) = LOWER(actual.sell_token_address) THEN TRUE
+            ELSE FALSE
+        END AS sell_token_address_test,
+
+        -- test the buy_token_address
+        CASE
+            WHEN LOWER(test.buy_token_address) = LOWER(actual.buy_token_address) THEN TRUE
+            ELSE FALSE
+        END AS buy_token_address_test,
+
+        -- test the token_pair
+        CASE
+            WHEN LOWER(test.token_pair) = LOWER(actual.token_pair) THEN TRUE
+            ELSE FALSE
+        END AS token_pair_test,
+
+        -- test the sell_amount
+        CASE 
+            WHEN ROUND(test.sell_amount, 2) = ROUND(actual.sell_amount, 2) THEN TRUE
+            ELSE FALSE
+        END AS sell_amount_test,
+
+        -- test the buy_amount
+        CASE 
+            WHEN ROUND(test.buy_amount, 2) = ROUND(actual.buy_amount, 2) THEN TRUE
+            ELSE FALSE
+        END AS buy_amount_test,
+
+        -- test the sell_amount_raw
+        CASE 
+            WHEN test.sell_amount_raw = actual.sell_amount_raw THEN TRUE
+            ELSE FALSE
+        END AS sell_amount_raw_test,
+
+        -- test the buy_amount_raw
+        CASE 
+            WHEN test.buy_amount_raw = actual.buy_amount_raw THEN TRUE
+            ELSE FALSE
+        END AS buy_amount_raw_test,
```

Comment:
There may be type parsing error in comparison when you compare `*_amount_raw`:
https://dune.com/queries/2267897?d=1

Reply:
👍